src/sw.test.js

```diff
@@ -28,26 +28,18 @@ describe('Service Worker', () => {
     const TEST_IMAGE_URL = 'https://ichef.test.bbci.co.uk';
     const BASE_IMAGE_URL = 'https://ichef.bbci.co.uk';
 
-    describe('image requested', () => {
+    describe('image requested when webp not supported', () => {
       it.each`
-        image                                           | expectedUrl
-        ${`${TEST_IMAGE_URL}/news/puppies.jpg`}         | ${`${TEST_IMAGE_URL}/news/puppies.jpg.webp`}
-        ${`${TEST_IMAGE_URL}/news/puppies.png`}         | ${`${TEST_IMAGE_URL}/news/puppies.png.webp`}
-        ${`${TEST_IMAGE_URL}/ace/standard/puppies.jpg`} | ${`${TEST_IMAGE_URL}/ace/standard/puppies.jpg.webp`}
-        ${`${TEST_IMAGE_URL}/ace/standard/puppies.png`} | ${`${TEST_IMAGE_URL}/ace/standard/puppies.png.webp`}
-        ${`${TEST_IMAGE_URL}/ace/ws/puppies.jpg`}       | ${`${TEST_IMAGE_URL}/ace/ws/puppies.jpg.webp`}
-        ${`${TEST_IMAGE_URL}/ace/ws/puppies.png`}       | ${`${TEST_IMAGE_URL}/ace/ws/puppies.png.webp`}
-        ${`${BASE_IMAGE_URL}/news/puppies.jpg`}         | ${`${BASE_IMAGE_URL}/news/puppies.jpg.webp`}
-        ${`${BASE_IMAGE_URL}/news/puppies.png`}         | ${`${BASE_IMAGE_URL}/news/puppies.png.webp`}
-        ${`${BASE_IMAGE_URL}/ace/standard/puppies.jpg`} | ${`${BASE_IMAGE_URL}/ace/standard/puppies.jpg.webp`}
-        ${`${BASE_IMAGE_URL}/ace/standard/puppies.png`} | ${`${BASE_IMAGE_URL}/ace/standard/puppies.png.webp`}
-        ${`${BASE_IMAGE_URL}/ace/ws/puppies.jpg`}       | ${`${BASE_IMAGE_URL}/ace/ws/puppies.jpg.webp`}
-        ${`${BASE_IMAGE_URL}/ace/ws/puppies.png`}       | ${`${BASE_IMAGE_URL}/ace/ws/puppies.png.webp`}
+        image                                        | expectedUrl
+        ${`${TEST_IMAGE_URL}/news/puppies.jpg.webp`} | ${`${TEST_IMAGE_URL}/news/puppies.jpg`}
+        ${`${BASE_IMAGE_URL}/news/puppies.jpg.webp`} | ${`${BASE_IMAGE_URL}/news/puppies.jpg`}
+        ${`${TEST_IMAGE_URL}/news/puppies.png.webp`} | ${`${TEST_IMAGE_URL}/news/puppies.png`}
+        ${`${BASE_IMAGE_URL}/news/puppies.png.webp`} | ${`${BASE_IMAGE_URL}/news/puppies.png`}
       `(`for $image is $expectedUrl`, async ({ image, expectedUrl }) => {
         ({ fetchEventHandler } = await import('./service-worker-test'));
 
         const event = {
-          request: new Request(image, { headers: { accept: 'webp' } }),
+          request: new Request(image, { headers: { accept: 'jpg' } }),
```

Comment:
Let's just remove the header (as this is likely to be the case in browsers that don't support `webp`)

```suggestion
          request: new Request(image),
```